src/pudl/transform/ferc1.py

```diff
@@ -198,13 +322,24 @@ def process_xbrl(
             )
         )
 
+    @cache_df(key="xbrl")
+    def wide_to_tidy_xbrl(self, df: pd.DataFrame) -> pd.DataFrame:
+        """Reshape the XBRL data from wide to tidy format.
+
+        This is a no-op in the abstract base class, but should be implemented for
+        child classes which need to reshape the XBRL data for concatenation with DBF.
+        Parameterization TBD based on additional experience. See:
+        https://github.com/catalyst-cooperative/pudl/issues/2012
+        """
+        return df
```

Comment:
I hope & expect that we will have real parameters for these methods, but I don't think we'll know what they should look like until after we've done several of these kinds of tables, and can see what different categories they fall into. So for now this is a totally bespoke, unparameterized transform that is nevertheless defined by the parent class so that it can be worked into the default pipeline.

I'd like to leave this as it is for now, rather than adding some meaningless parameterization boilerplate, and hopefully return to it later when we have more context.

It's also possible that each of these will just have to be unique, if there's a lot of structural variation.

Reply:
that is very fair and sounds like a great plan!